platforms/shared/tests/test-client/main.cpp

```diff
@@ -125,6 +127,25 @@ void initLogger()
     el::Loggers::reconfigureAllLoggers(conf);
 }
 
+void testUploadCount(prc::Client& client, prc::id_t accountId, prc::id_t instrumentId)
+{
+    prc::Counts counts;
+    time_point ftime = boost::chrono::system_clock::now() - boost::chrono::hours(1);
+
+    for (int i = 0; i < 3; ++i)
```

Comment:
I don't agree. This is code to test function works (see Web API example) and hardcoded values are just fine for it. It's neither unit test, nor integration test. It is more like smoke test.